### PR TITLE
[RFR] adding default sentry config & readme

### PR DESCRIPTION
**Pull request by openbook**
default config for sentry + some readmememe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +90,15 @@ elasticsearch:
     region: ''
 ```
 
+### Sentry logging
+
+Server side logging via [Sentry](http://sentry.io) can be enabled using the following configuration:
```
Comment (enginehouse) [inline README.md:95]: Dunno if this is important but should there be contact info regarding how to access the logs as they are password protected?

Comment (enginehouse) [reply README.md:95]: Guessing probably not....

Comment (openbook) [reply README.md:95]: I think @Koded wanted to keep all keys out of the repo (which I already committed once and then removed 😬 )

Comment (enginehouse) [reply README.md:95]: Fully - just wondered if there needed to be a named contact person to administer this - but probs not : )))

**Pull request merged**